container-storage-setup.sh

```diff
@@ -958,13 +958,27 @@ create_disk_partitions() {
   local devs="$1" part
 
   for dev in $devs; do
+    # wipefs /dev/disk does not wipe any lvm signatures which might be
+    # present on /dev/diskpart1. This signature will become visible to
+    # lvm udev rules and will kickstart volume creation as soon as partion
+    # is created and race with further partition commands like wipefs,
+    # pvcreate etc. So zero out first few MB of disk in an attempt to
+    # wipe any lvm signatures on first partition.
+    #
+    # By now we have ownership of disk and we have checked there are no
+    # signatures on disk or signatures have been wiped. Dont care about
+    # any signatures now on in the middle of disk.
+    Info "Writing zeros to first 4MB of device $dev"
+    if ! dd if=/dev/zero of=$dev bs=1M count=4; then
```

Comment:
(Is there a possible quoting problem with $dev?  Are we guaranteed only ever to have vanilla space-non-containing names?)

Reply:
can device path container space? I assumed it can't and that's why did not bother to put double quotes.